package.json

```diff
@@ -19,7 +19,8 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint ./src --ext .js,.jsx",
-    "lint:fix": "eslint ./src --ext .js,.jsx --fix"
+    "lint:fix": "eslint ./src --ext .js,.jsx --fix",
+    "coverage": "react-scripts test --coverage"
```

Comment:
This looks good. You could have also modified the line 19 and add the `--coverage`. In my experience, the rule for a project is: force the coverage and avoid it, so you don't need to have both options, but this is ok